homepage.css

```diff
@@ -0,0 +1,60 @@
+.nav-container{
+    padding: 0 var(--xs);
+    flex-wrap: wrap;
```

Comment:
Just a small suggestion that you can have this property in your utils like this will be widely used throughout your app and future projects

Reply:
Sure